examples/protobuf_parser.py

```diff
@@ -97,4 +99,15 @@
   repeated Person person = 1;
 }"""
 
-parser.runTests([test1, test2])
```

Comment:
And added a test too, thanks!

Reply:
Nothing to do, you missed my changes at L113
```suggestion
parser.runTests([test1, test2, test3])
```